src/Servers/Kestrel/Core/src/Features/WebTransportStreamType.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Features;
+
+/// <summary>
+/// Represents the different types of WebTransport streams.
+/// </summary>
+public enum WebTransportStreamType
```

Comment:
You are right, this should be internal. It was public because a previous version of my implementation would return it to the application. However, now we just use the default `Stream.CanRead` and `Stream.CanWrite`. So, there is no need to expose it to the public.